state/azure/cosmosdb/cosmosdb.go

```diff
@@ -43,16 +42,6 @@ type CosmosItem struct {
 	PartitionKey string      `json:"partitionKey"`
 }
 
-// CosmosItemWithRawMessage is a version of CosmosItem with a Value of RawMessage so this field
-// is not marshalled.  If it is marshalled it will end up in a form that
-// cannot be unmarshalled.  This type is used when SetRequest.Value arrives as bytes.
-type CosmosItemWithRawMessage struct {
```

Comment:
Using a separate type would complicated the code that handles state transactions. Turns out we don't need a separate type for this case, because `CosmosItem.Value` is already an `interface{}`

Reply:
Is this a backwards compatible change in the sense that data stored with the old code can be read with this version?

Reply:
So what happens with the bug is that your content is stored as a base64 encoded JSON string. So if the scenario you're talking about is using the SDK you can't round-trip that data because deserialization would fail.

If someone used the generated gRPC client to retrieve the data and manually deserialize it from the base64 string, then yes, this would back that use case.

🌈 Every change is a breaking change to someone 🌈

Reply:
OK. LGTM.